holidays/holiday_groups.py

```diff
@@ -684,6 +684,16 @@ def _add_world_war_two_victory_day(self, holiday_name):
         """
         return self._add_holiday(holiday_name, MAY, 9)
 
+    def _add_remembrance_day(self, holiday_name):
```

Comment:
In general I add a new method for a holiday when at least 2 countries share it.
If you're going to keep it please move it right below the `def _add_new_years_eve`

Reply:
> In general I add a new method for a holiday when at least 2 countries share it.

Armistice Day also have France, USA, Belgium and Serbia.

> If you're going to keep it please move it right below the `def _add_new_years_eve`

OK, but I don't quite understand the principle of order. :)